runtime/onert/core/include/backend/IPortableTensor.h

```diff
@@ -37,12 +38,18 @@ namespace backend
 class IPortableTensor : public ITensor
 {
 public:
+  IPortableTensor(const ir::OperandInfo &info) : _info(info) {}
+
   virtual ~IPortableTensor();
   virtual const ir::Sparsity *sparsity() const { return nullptr; }
+  const ir::OperandInfo &get_info() const { return _info; }
```

Comment:
As you can see I moved _info field from derived classes here.
I suppose this is fine, but I need to verify it with code owner, I hope I added right persons into review =)

Whole idea of this changes is: many calls to small virtual methods are expensive, and this PR removes calls to `ITensor::dimension` and `ITensor::num_dimensions` in cpu_backend.